specs/prod-genomes.spec

```diff
@@ -186,190 +211,298 @@ module KBaseGenomes {
+            CDS. The first element in the tuple corresponds to the "contig_id",
+            such as "NC_000913.3". The second element in the tuple is an index in
+            the contig of where the sequence starts. The third element is either a
+            plus or minus sign indicating whether it is on the 5' to 3' leading
+            strand ("+") or on the 3' to 5' lagging strand ("-"). The last element
+            is the length of the sub-sequence.
+            For a location on the leading strand (denoted by "+"), the index is
+            of the leftmost base, and the sequence extends to the right. For a
+            location on the lagging strand (denoted by "-"), the index is of
+            the rightmost base, and the sequence extends to the left.
+            NOTE: the last element in each tuple is the *length* of each
+            sub-sequence. If you have a location such as ("xyz", 100, "+", 50),
+            then your sequence will go from index 100 to index 149 (this has a
+            length of 50). It *does not* go from index 100 to index 150, as
+            that would have a length of 51.
+            Likewise, if you have the location ("xyz", 100, "-", 50), then the
+            sequence extends from 100 down to 51, which has a length of 50
+            bases. It does not go from index 100 to 50, as that would have a
+            length of 51.
+        md5 - string - md5 of the dna sequence - TODO clarification
+        protein_md5 - string - hash of the protein sequence that this CDS encodes
+        parent_gene - string - gene (feature) from which this CDS comes from,
+            including introns and UTRs that have been removed to create this CDS.
+        parent_mrna - string - mRNA sequence from which this sequence is derived,
+            including UTRs but not introns.
+        note - string - TODO
+        functions - list<string> - list of protein products or chemical
+            processes that this sequence creates, facilitates, or influences.
+        functional_descriptions - list<string> - TODO list of protein products or chemical
+            processes that sequence creates, facilitates, or influences.
+        ontology_terms - mapping<string, mapping<string, list<int>>> - a mapping
+            of ontology source id (eg. "GO") to a mapping of term IDs (eg "GO:16209")
+            to a list of indexes into the ontology_events data (found in the top
+            level of the genome object). The index into an ontology event indicates
+            what service and method created this term assignment.
+        flags - list<string>  - (controlled vocab) fields from the genbank source. A
+            common example is "pseudo" for pseudo-genes that do not encode proteins,
+            which shows up as "/pseudo" in the genbank.
+            Values can be: "pseudo", "ribosomal_slippage", "trans_splicing"
+        warnings - list<string> - TODO
```

Comment:
"list of plain text warnings related to the importing or construction of the feature."